site/app/controllers/banner/BannerController.php

```diff
@@ -0,0 +1,201 @@
+        }
+
+        for ($j = 0; $j < $count_item; $j++) {
+            $all_match = false;
+            if ($uploaded_files['name'][$j] === $extra_name) {
+                $all_match = true;
+            }
+
+            if (is_uploaded_file($uploaded_files["tmp_name"][$j])) {
+                $dst = FileUtils::joinPaths($full_path, $uploaded_files["name"][$j]);
+
+                if (!$all_match) {
+                    [$width, $height] = getimagesize($uploaded_files["tmp_name"][$j]);
+                    if ($width > 800 || $height > 100) {
+                        return JsonResponse::getErrorResponse("File dimensions must be no more than 800x100 pixels.");
+                    }
+                    if ($width < 200 || $height < 10) {
+                        return JsonResponse::getErrorResponse("File dimensions must be no less than 200x10 pixels.");
+                    }
+                }
+
+                if (strlen($dst) > 255) {
+                    return JsonResponse::getErrorResponse("Path cannot have a string length of more than 255 chars.");
+                }
+
+                if (!@copy($uploaded_files["tmp_name"][$j], $dst)) {
+                    return JsonResponse::getErrorResponse("Failed to copy uploaded file '{$uploaded_files['name'][$j]}' to current location.");
+                }
+            }
+            else {
+                return JsonResponse::getErrorResponse("The temporary file '{$uploaded_files['name'][$j]}' was not properly uploaded.");
+            }
+
+            if (!@unlink($uploaded_files["tmp_name"][$j])) {
+                return JsonResponse::getErrorResponse("Failed to delete the uploaded file '{$uploaded_files['name'][$j]}' from temporary storage.");
+            }
+
+            if ($all_match) {
+                continue;
+            }
+            $community_event_image = new BannerImage(
+                $specificPath,
+                $actual_banner_name,
+                $extra_name,
+                $link_name,
+                $release_date,
+                $close_date,
+                $folder_made_name
+            );
+            $this->core->getSubmittyEntityManager()->persist($community_event_image);
+        }
+        $this->core->getSubmittyEntityManager()->flush();
+        return JsonResponse::getSuccessResponse("Successfully uploaded!");
+    }
+
+    /**
+     * @Route("/banner/delete", methods={"POST"})
+     */
+    public function ajaxDeleteEventFiles(): JsonResponse {
+        $entity_manager = $this->core->getSubmittyEntityManager();
+        $event_repository = $entity_manager->getRepository(BannerImage::class);
+        $event_items = $event_repository->findBy(['name' => $_POST['name'] ]);
+        if (count($event_items) === 0) {
+            $error_message = "Banner item with name '" . $_POST['name'] . "' not found in the database.";
+            return JsonResponse::getErrorResponse($error_message);
+        }
+
+        $event_item = $event_items[0];
```

Comment:
You could use findOneBy instead. Also it would probably be better to delete by ID rather than name.